src/codegen/codegen.h

```diff
@@ -1,6 +1,10 @@
 #ifndef TACO_CODEGEN_H
 #define TACO_CODEGEN_H
 
+#define PRINT_FUNC 0
+#define PRINT_MEM_HOST_TO_DEV 1
+#define PRINT_MEM_DEV_TO_HOST 2
```

Comment:
Should replace these with an enum.

Reply:
Oh, I see. I will replace them with an enum.